options.cpp

```diff
@@ -217,6 +217,8 @@ options::options()
        ("spindown-time", po::value<Time>(), "time required to the spindle to return to 0 rpm")
        ("mill-feed", po::value<Velocity>(), "feed while isolating in [i/m] or [mm/m]")
        ("mill-vertfeed", po::value<Velocity>(), "vertical feed while isolating in [i/m] or [mm/m]")
+       ("x-offset", po::value<double>()->default_value(0.0), "distance between the PCB origin and the machine origin in the X axe")
```

Comment:
This should not have type "double", it should be a "Length".  Same for y-offset below.

After fixing that, it will clean up the cfactor stuff above.

With this fixed, it will be possible to write `x-offset=10mm`, for example.